lib/utils/build-envs.js

```diff
@@ -0,0 +1,22 @@
+import assign from 'lodash/assign';
+
+/**
+ * Returns the default and user specified environment variables.
+ *
+ * @private
+ * @method buildEnvs
+ * @param command {Command} Used to set the default values coming from the specified command
+ * @param [project] {Project} Used to set the environment of the project
+ */
+export default function buildEnvs(command, project) {
```

Comment:
This feels like it belongs on either the Command base class, or on the Server & Test command subclasses. I can't really see it being used anywhere else, so lib/utils seems like a slightly too generic home for this code.